README.md

```diff
@@ -26,6 +26,17 @@ Or install it yourself as:
 $ gem install ratelimit
 ```
 
+## Setup
+
+By default, Ratelimit will try to connect to a Redis server on `127.0.0.1:6379`. But, in most scenarios, you need to specify a different configuration.
+To do that, create a new redis client (or use an existing one) pointing to the right server and pass it as a parameter in the constructor.
+
+``` rb
+redis = Redis.new(host: "10.0.1.1", port: 6380, db: 15)
+...
+ratelimit = Ratelimit.new("messages", redis: redis)
+```
+
```

Comment:
I think for me this feels very prominent up here. I think I would reserve the first words for an actual usage example. Can you move the section maybe into the usage section under `### Connecting to an external redis instance`  or something?

Reply:
Sure! I'll move it into the usage section.